[0.035s] Invoking command in '/home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer': CONDA_PROMPT_MODIFIER=(adaface) /home/minha/anaconda3/envs/adaface/bin/cmake /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer -DCMAKE_INSTALL_PREFIX=/home/minha/myagv_ros2/myagv_ros2/install/myagv_cartographer
[0.080s] -- Found ament_cmake: 1.3.8 (/opt/ros/humble/share/ament_cmake/cmake)
[0.166s] -- Found ament_lint_auto: 0.12.10 (/opt/ros/humble/share/ament_lint_auto/cmake)
[0.229s] Error parsing '/home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml':
[0.229s] Traceback (most recent call last):
[0.229s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/site-packages/catkin_pkg/package.py", line 610, in parse_package_string
[0.229s]     root = dom.parseString(data)
[0.229s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/minidom.py", line 1998, in parseString
[0.229s]     return expatbuilder.parseString(string)
[0.229s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/expatbuilder.py", line 925, in parseString
[0.230s]     return builder.parseString(string)
[0.230s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/xml/dom/expatbuilder.py", line 223, in parseString
[0.230s]     parser.Parse(string, True)
[0.230s] xml.parsers.expat.ExpatError: mismatched tag: line 12, column 33
[0.230s] 
[0.230s] During handling of the above exception, another exception occurred:
[0.230s] 
[0.230s] Traceback (most recent call last):
[0.230s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 150, in <module>
[0.230s]     main()
[0.230s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 53, in main
[0.230s]     raise e
[0.231s]   File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 49, in main
[0.231s]     package = parse_package_string(
[0.231s]   File "/home/minha/anaconda3/envs/adaface/lib/python3.9/site-packages/catkin_pkg/package.py", line 612, in parse_package_string
[0.231s]     raise InvalidPackage('The manifest contains invalid XML:\n%s' % ex, filename)
[0.231s] catkin_pkg.package.InvalidPackage: Error(s) in package '/home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml':
[0.231s] The manifest contains invalid XML:
[0.231s] mismatched tag: line 12, column 33
[0.235s] [31mCMake Error at /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:95 (message):
[0.236s]   execute_process(/home/minha/anaconda3/envs/adaface/bin/python3
[0.236s]   /opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py
[0.236s]   /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer/package.xml
[0.236s]   /home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer/ament_cmake_core/package.cmake)
[0.236s]   returned error code 1
[0.236s] Call Stack (most recent call first):
[0.236s]   /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:49 (_ament_package_xml)
[0.236s]   /opt/ros/humble/share/ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake:31 (ament_package_xml)
[0.236s]   CMakeLists.txt:26 (ament_lint_auto_find_test_dependencies)
[0.236s] 
[0.236s] [0m
[0.236s] -- Configuring incomplete, errors occurred!
[0.374s] Invoked command in '/home/minha/myagv_ros2/myagv_ros2/build/myagv_cartographer' returned '1': CONDA_PROMPT_MODIFIER=(adaface) /home/minha/anaconda3/envs/adaface/bin/cmake /home/minha/myagv_ros2/myagv_ros2/src/myagv_cartographer -DCMAKE_INSTALL_PREFIX=/home/minha/myagv_ros2/myagv_ros2/install/myagv_cartographer
